packages/agoric-cli/test/snapshots/test-inter-cli.js.md

```diff
@@ -119,7 +126,7 @@ Generated by [AVA](https://avajs.dev).
     add collateral to the reserve␊
     ␊
     Options:␊
-      --giveCollateral [number]   Collateral to give␊
-      --collateralBrand [string]  Collateral brand key (default: "IbcATOM")␊
-      --offerId [number]          Offer id (default: "bid-978307200000")␊
-      -h, --help                  display help for command`
+      --give <number>              Collateral to give␊
+      --collateral-brand <string>  Collateral brand key (default: "IbcATOM")␊
```

Comment:
not certain about this style change. I guess it's more CLI-y

Reply:
Dean suggested it to match cosmos-sdk CLI style.